package.json

```diff
@@ -54,6 +48,11 @@
 		"storybook": "start-storybook  -c ./storybook -p 6006 --ci",
 		"storybook:build": "build-storybook  -c ./storybook -o ./storybook/dist",
 		"test": "wp-scripts test-unit-js --config tests/js/jest.config.json",
+		"test:e2e:down": "docker-compose down",
+		"test:e2e:down:reset": "docker-compose down -v",
+		"test:e2e:ssh": "docker exec -it woo-blocks_wordpress-www_1 /bin/bash",
+		"test:e2e:up": "docker-compose up -d",
+		"test:e2e:up:build": "docker-compose up --build -d",
 		"test:e2e": "./tests/bin/e2e-test-integration.js",
```

Comment:
When I mentioned chaining I meant here; we can run the docker up commands too so you don't need to manually use the commands above.

Reply:
depending on if you're testing things, you would run `test:e2e:up` once but would run  `test:e2e` many times.

Reply:
Yep exactly @senadir. Also there may need to be a long wait between `up` and running the first test; and the user needs to know to `down` when they are finished as well. 

Having said that I agree with @mikejolley we should make this as simple as possible to run – let's keep iterating on it and polishing the developer experience :)